lib/repo/repository.go

```diff
@@ -19,6 +28,15 @@ import (
 // S3Bucket keeps default S3 bucket path
 const S3Bucket = "https://cdn.isaax.io/isaax-distro/versions.json"
 
+// IoTIt default repo path
```

Comment:
__[golint]__
* [golint] comment on exported const IoTItRepo should be of the form "IoTItRepo ..."

<!-- Comment by [SideCI](https://sideci.com) -->